OutOfSchool/OutOfSchool.WebApi/Controllers/StatisticController.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Localization;
+using OutOfSchool.WebApi.Services;
+
+namespace OutOfSchool.WebApi.Controllers
+{
+    /// <summary>
+    /// Controller with operations to get popular workshops and categories.
+    /// </summary>
+    [Route("[controller]/[action]")]
+    [ApiController]
+    [Authorize(AuthenticationSchemes = "Bearer")]
+
+    public class StatisticController : ControllerBase
+    {
+        private readonly IStatisticService service;
+        private readonly IStringLocalizer<SharedResource> localizer;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="StatisticController"/> class.
+        /// </summary>
+        /// <param name="service">Service to get statistic.</param>
+        /// <param name="localizer">Localizer.</param>
+        public StatisticController(IStatisticService service, IStringLocalizer<SharedResource> localizer)
+        {
+            this.service = service;
+            this.localizer = localizer;
+        }
+
+        /// <summary>
+        /// Get popular categories.
+        /// </summary>
+        /// <param name="number">The number of entries.</param>
+        /// <returns>List of popular categories.</returns>
+        [HttpGet("{number}")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
```

Comment:
For better swagger documentation when you use `Task<IActionResult>` you need to specify the object type in this attribute like `[ProducesResponseType(StatusCodes.Status200OK, Type = typeof(YOUR_DTO_HERE))]`